0-ad-history-mod-package/simulation/templates/structures/ptol/civil_centre.xml

```diff
@@ -7,6 +7,11 @@
   <Identity>
     <Civ>ptol</Civ>
     <SpecificName>Agora</SpecificName>
+    <History>
+        The Ptolemaic Kingdom was, just as Macedon, ruled by a monarchy. The royal bloodline descended from Ptolemy I Soter, the founder. Monarchs held both the Egyptian title of pharaoh as well as the Greek of “Basileus” (king). 
+        Below the king, there was pyramidic administration and bureaucracy system in place adopted from Greece: from a central government at the top to regional ministers to local village chiefs at the bottom. Most political offices were carried out by Greeks. 
+        This is because, from a legal perspective society was split into two classes, with the upper class being the Greeks and the lower the native Egyptians. There were also, for example, different courts for the two classes.
```

Comment:
```suggestion
        This is because, from a legal perspective, society was split into two classes, with the upper class being the Greeks and the lower the native Egyptians. There were also, for example, different courts for the two classes.
```